# List of all AprilTag detections from a single camera frame.
Header header
ObjectDetection[] detections
